### DISCUSSION TITLE
maybe it's a bug --use lora in ADetailer prompt

### DISCUSSION THREAD

**Discussion by ShaoYiSky117** [Bing-su/adetailer#247] [Q&A]
my webui  v1.5.1
my adetailer  v23.7.10
my a1111-sd-webui-lycoris v8e97bf5 [https://github.com/KohakuBlueleaf/a1111-sd-webui-lycoris]
All are the latest versions

bug:
when I use [LyCORIS] in the main prompt , if i use <lyco:XXXX: 1>,It will make the Lora in the prompt word of ADetailer ineffective.

when I use [LyCORIS] in the main prompt, but if i use <lora:XXXX: 1>,the Lora in the prompt word of ADetailer effective.

this issue appears in the latest version and has not appeared before the 1.5.1 version of webui.

I want to know if the<lyco: XXXX: 1>syntax has been deprecated in the new version of adetailer? Would you still consider solving this problem?

Thanks!

**Comment by Bing-su** [comment]
adetailer는 controlnet을 제외한 다른 어떤 확장의 기능을 건들지 않습니다. lycoris쪽에 문의해보시는게 나을 것 같습니다.

**Comment by ShaoYiSky117** [comment]
Thanks!
